modules/core/src/main/java/org/apache/ignite/internal/commandline/management/ManagementCommands.java

```diff
@@ -151,11 +158,11 @@ private void printHelp(Logger log) {
 
                 while (argIter.hasNextSubArg()) {
                     ManagementURLCommandArg uriArg = CommandArgUtils.of(
-                        argIter.nextArg("Expected one of uri arguments"), ManagementURLCommandArg.class
+                        argIter.nextArg("Invalid uri arguments"), ManagementURLCommandArg.class
                     );
 
                     if (uriArg == null)
-                        throw new IllegalArgumentException("Expected one of auto-adjust arguments");
+                        throw new IllegalArgumentException("Invalid uri arguments");
```

Comment:
Can we extend the message to highlight what exact arg is _null_?

Reply:
Done